test/extended/cluster/utils.go

```diff
@@ -519,3 +519,39 @@ func SetNamespaceLabels(c kclientset.Interface, name string, labels map[string]s
 	ns.Labels = labels
 	return c.CoreV1().Namespaces().Update(ns)
 }
+
+func ProjectExists(oc *exutil.CLI, name string) (bool, error) {
+	projectList, err := oc.AdminProjectClient().Project().Projects().List(metav1.ListOptions{})
```

Comment:
Wouldn't it make more sense to do a `Get` instead? That way there's one call and one check vs iteration over the entire returned list?

Reply:
Done.
It also raises an error when the project does not exist.
So change the logic to error-occurring = project-not-existing.